query-node/schemas/contentNft.graphql

```diff
@@ -263,3 +269,17 @@ type Bid @entity {
 }
 
 # TODO entity for (cancelable) offers; will be needed to see history of offers
+
+type CuratorAgentPermissions @entity {
+  "Relevant channel"
+  curatorGroup: CuratorGroup!
+
+  "Related member"
+  curator: Curator!
+
+  # unfortunately permissions can't be modeled as list of enums
+  # `permissions: [ChannelActionPermission!]!`
+  # rework it after this feature is available https://github.com/Joystream/hydra/issues/507
+  "List of member's permissions"
+  permission: ChannelActionPermission!
```

Comment:
Same as suggested above. We can have permissions as a list of Strings, e.g., permissions: [String!]!.

Reply:
Done